tests/api/entities/languages.test.js

```diff
@@ -0,0 +1,50 @@
+import 'should'
+import { publicReq } from '#tests/api/utils/utils'
+
+const endpoint = '/api/entities?action=languages'
+
+// The following tests require to have language entities indexed locally.
+// This can be done by running `npm run indexation:wikidata:load-languages-from-query`
+describe('entities:languages:search', () => {
+  it('should find a language by its label in the requested language', async () => {
+    const res = await publicReq('get', `${endpoint}&search=Korsische&lang=de`)
+    res.languages.should.be.an.Array()
+    const language = res.languages[0]
+    language.uri.should.equal('wd:Q33111')
+  })
+
+  it('should find a language by its label in another language', async () => {
+    const res = await publicReq('get', `${endpoint}&search=Korsische&lang=fr`)
```

Comment:
just by looking at those test, ive difficulty to grasp what does the `lang` parameter brings to the request. I think it whould make since clearer to have a "negative" test which would document what not having `lang` param does.

Reply:
The `lang` parameter should just boost some results if they happen to match on the requested language. The negative test would get us deep in the questions of Elasticsearch scoring; I would rather not go there, as that seems to be a receipt for a fragile test.

Reply:
Would something like:

```js
it('should find english as default language', async () => {
  const res = await publicReq('get', `${endpoint}&search=Italian`)
  const language = res.languages[0]
  language.uri.should.equal('wd:Q652')
})
```

be subjected to results that would be inconsistent/not future proof ?

This test would not document the purpose of this `lang` parameter (thus digging in ES scoring), but at least we can know that it's optional (?)

Also we could include your description:

> The `lang` parameter should just boost some results if they happen to match on the requested language. 

in a top comment of the test file.